metricflow/sql/render/trino.py

```diff
@@ -45,7 +46,7 @@ def visit_generate_uuid_expr(self, node: SqlGenerateUuidExpression) -> SqlExpres
         )
 
     @override
-    def visit_time_delta_expr(self, node: SqlSubtractTimeIntervalExpression) -> SqlExpressionRenderResult:
+    def visit_subtract_time_interval_expr(self, node: SqlSubtractTimeIntervalExpression) -> SqlExpressionRenderResult:
```

Comment:
Would it have been more intuitive if we just made it so that we only had `visit_time_delta` and the direction just can be done via supplying it a positive or negative count?

Reply:
ooo gotcha!